ci/fpm-deployment.sh

```diff
@@ -11,6 +11,7 @@ fypp="${FYPP:-$(which fypp)}"
 # Arguments for the fypp preprocessor
 maxrank=4
 fyflags="${FYFLAGS:--DMAXRANK=$maxrank}"
+fyflags="${fyflags} -DFPM_DEPLOYMENT=True"
```

Comment:
this directive should be documented somewhere. Currently, it is fuzy to me what it does.

Reply:
Thanks @jvdp1, this is not needed anymore since we left backend-vs-external library to be resolved by preprocessor directives. Removed in [592e085](https://github.com/fortran-lang/stdlib/pull/772/commits/592e0852d2de09f6e5d5488205002c97d301f96c)